app/models/article.rb

```diff
@@ -355,13 +353,11 @@ def cloudinary_video_url
   end
 
   def video_duration_in_minutes
-    minutes = video_duration_in_minutes_integer
-    seconds = video_duration_in_seconds.to_i % 60
-    seconds = "0#{seconds}" if seconds.to_s.size == 1
+    duration = ActiveSupport::Duration.build(video_duration_in_seconds.to_i).parts
+    minutes_and_seconds = format("%<minutes>02d:%<seconds>02d", duration)
+    return minutes_and_seconds if duration[:hours] < 1
 
-    hours = (video_duration_in_seconds.to_i / 3600)
-    minutes = "0#{minutes}" if hours.positive? && minutes < 10
-    hours < 1 ? "#{minutes}:#{seconds}" : "#{hours}:#{minutes}:#{seconds}"
+    "#{duration[:hours]}:#{minutes_and_seconds}"
   end
 
   def video_duration_in_minutes_integer
```

Comment:
can we move this in the serializer? It's only called there... The serializer already contains some serialization logic.

Reply:
Since this got merged already, I'll check that tomorrow.